scripts/build_shapes.py

```diff
@@ -105,7 +105,19 @@ def download_GADM(country_code, update=False, out_logging=False):
         #  create data/osm directory
         os.makedirs(os.path.dirname(GADM_inputfile_gpkg), exist_ok=True)
 
-        with requests.get(GADM_url, stream=True) as r:
+        try:
+            r = requests.get(GADM_url, stream=True, timeout=300)
+        except (requests.exceptions.ConnectionError, requests.exceptions.Timeout):
+            raise Exception(
+                f"GADM server is down at {GADM_url}. Data needed for building shapes can't be extracted.\n\r"
+            )
+        except Exception as exception:
+            raise Exception(
+                f"An error happened when trying to load GADM data by {GADM_url}.\n\r"
+                + str(exception)
```

Comment:
Here you used also the option "exc_info" as an alternative, you mentioned it was nice.
Should we go instead with the current implementation?

Reply:
Yeah, absolutely, it's convenient to use `exc_info` is king of an argument in `logger.error()`, but `Exception` doesn't agree to accept `exc_info` as an argument. 

It's possible to call `sys.exc_info()`, but it looks less convenient as compared with `str(exception)`. Do I miss anything? :)

Reply:
It is good the way it is, ready to merge :)